sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/models/BaseOptions.java

```diff
@@ -18,14 +18,14 @@
     public static final int NO_CACHE = 0;
 
     /**
-     * Default caching value of one hour.
+     * Default caching value of one day.
      */
-    public static final int DEFAULT_ONE_HOUR = 60 * 60;
+    public static final int DEFAULT_EXPIRATION = 60 * 60 * 24;
```

Comment:
When we don't use the default java time unit (millis) we should have `_SECONDS` or `_IN_SECONDS` in constant name.